### PR TITLE
{data}[intel/2020a] NCL v6.6.2 and dependencies

### PR DESCRIPTION
**Pull request by SimonPinches**

### CODE REVIEW

File: easybuild/easyconfigs/g/g2lib/g2lib-3.1.0-intel-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name = 'g2lib'
+version = '3.1.0'
+
+homepage = 'http://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/'
```
Comment (Micket) [inline easybuild/easyconfigs/g/g2lib/g2lib-3.1.0-intel-2020a.eb:4]:
```suggestion
homepage = 'https://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/'
```

**Review by Micket — DISMISSED** [review]

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 19 out of 19 (7 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.8.2003, x86_64, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/5b414b7300ac80e21da175d1ea6d3082 for a full test report.

**Comment by Micket** [comment]
retrying github actions

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 9 out of 15 (7 easyconfigs in this PR)
node305.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/6761ce639099e6a24db7ada7e3928980 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 8 out of 8 (7 easyconfigs in this PR)
node355.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/bac1c5dba31ca812a879d396fca03c5a for a full test report.

**Review by lexming — CHANGES_REQUESTED** [review]
The build works well. However there are a few easyconfigs that do not need to use the `intel` toolchain and can be moved to `GCCcore`, as it is the case with their previous versions. Please see below:
* `g2clib-1.6.0-intel-2020a.eb ` should be `g2clib-1.6.0-GCCcore-9.3.0.eb`. We already have `g2clib-1.6.0-GCCcore-8.3.0.eb` that you can use as reference.
* `g2lib-3.1.0-intel-2020a.eb` should be `g2lib-3.1.0-GCCcore-9.3.0.eb`. We already have `g2lib-3.1.0-GCCcore-8.3.0.eb` that you can use as reference, you will see that `buildopts` is slightly different to yours.
* `UDUNITS-2.2.26-intel-2020a.eb` is not needed. We already have `UDUNITS-2.2.26-GCCcore-9.3.0.eb`, which will be picked up by `NCL-6.6.2-intel-2020a.eb`. You can remove the easyconfig for UDUNITS from this PR and leave the dependency line in NCL as it is.

**Comment by SimonPinches** [comment]
Thanks for the push @lexming.  I was being lazy when I saw that `g2lib` needed patching to work with `GCCcore/9.3.0`.  All done now.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node368.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c1960cac3ac31ce66e35bf3e1f4a7382 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 14 out of 14 (6 easyconfigs in this PR)
node127.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/09132e74dd7a21705788e92f6b3359e9 for a full test report.

**Review by lexming — APPROVED** [review]
@SimonPinches Thanks for the update!

LGTM

**Comment by lexming** [comment]
Going in, thanks @SimonPinches !

**Pull request merged**